scapy/error.py

```diff
@@ -64,13 +64,13 @@ def emit(self, record):
 log_loading = logging.getLogger("scapy.loading")          # logs when loading Scapy
 
 
-def warning(x, *args, **kargs):
+def warning(x, onlyOnce=False, *args, **kargs):
     """
     Prints a warning during runtime.
 
     onlyOnce - if True, the warning will never be printed again.
     """ 
-    if kargs.pop("onlyOnce", False):
+    if kargs.pop("onlyOnce", False) or onlyOnce:
```

Comment:
What is the purpose of this change? Also, if you explicitly put onlyOnce in the arguments, it cannot be in kwargs.

Reply:
Right, I'll do that 
That's currently a bug while scapy calls ˋwarning("msg", True)` since **kwargs were added to the warning function. But now kargs.pop is useless